### DISCUSSION TITLE
On multi-site installs, show revision history per site, not per entry

### DISCUSSION THREAD

**Discussion by caseyreid** [craftcms/cms#8912] [Ideas]
On Craft 2 sites with multiple locales, the revision history for an entry would be isolated to the locale. If I made a change to an entry in Locale A, that revision would not show in the revision history for Locale B, and vice-versa.

This was useful for content editors, in that they could see if a specific locale of an entry had recent revisions, and could even revert back to those revisions, per locale.

On Craft 3 multi-site setups, the revision history seems to be tied to the entry, not the site. If I make an edit to an entry in Site A, that revision will show in Site B's revision history. In addition, there is no indication of what site the entry revision was made in.

I'd like to request that revisions be saved per site on multi-site setups.

If the revision history needs to stay as is, per entry, then I'd like to request adding the site name to the revision. As an example:

![Revision History](https://user-images.githubusercontent.com/133320/91755564-f2d3b480-eb90-11ea-8ad2-a42d715efc78.png)

**Comment by brandonkelly** [comment]
I don’t think we would fully remove them, because all fields could potentially have been updated across all sites whenever an entry is saved, depending on their translation methods. Plus drafts are multi-site, so if the revision came from a draft, then it’s possible that multiple sites’ content were directly updated by the draft simultaneously.

So I think at best we could start showing the site that the entry was saved for, if not being saved via a draft.
